### PR TITLE
Add client_max_body_size config

### LINKED ISSUE #1: api: nginx content body size sometimes not large enough
Happened to find some failures in nginx logs:
```
2020/06/25 16:15:33 [error] 3336#0: *27331 client intended to send too large body: 1631835 bytes, client: 10.168.20.167, server: ara.domain.tld, request: "POST /api/v1/results HTTP/1.1", host: "ara.domain.tld"
2020/06/25 16:15:34 [error] 3336#0: *27332 client intended to send too large body: 1565451 bytes, client: 10.168.20.167, server: ara.domain.tld, request: "POST /api/v1/results HTTP/1.1", host: "ara.domain.tld"
2020/06/25 16:23:07 [error] 3336#0: *28184 client intended to send too large body: 1698120 bytes, client: 10.168.20.167, server: ara.domain.tld, request: "POST /api/v1/results HTTP/1.1", host: "ara.domain.tld"
2020/06/25 16:23:09 [error] 3336#0: *28185 client intended to send too large body: 1641727 bytes, client: 10.168.20.167, server: ara.domain.tld, request: "POST /api/v1/results HTTP/1.1", host: "ara.domain.tld"
2020/06/25 16:23:09 [error] 3336#0: *28186 client intended to send too large body: 1549986 bytes, client: 10.168.20.167, server: ara.domain.tld, request: "POST /api/v1/results HTTP/1.1", host: "ara.domain.tld"
2020/06/26 19:40:48 [error] 3336#0: *34997 client intended to send too large body: 1072237 bytes, client: 10.168.43.105, server: ara.domain.tld, request: "POST /api/v1/results HTTP/1.1", host: "ara.domain.tld"
2020/06/26 19:40:49 [error] 3336#0: *34999 client intended to send too large body: 1124136 bytes, client: 10.168.43.105, server: ara.domain.tld, request: "POST /api/v1/results HTTP/1.1", host: "ara.domain.tld"
2020/06/26 19:40:52 [error] 3336#0: *35000 client intended to send too large body: 1149775 bytes, client: 10.168.43.105, server: ara.domain.tld, request: "POST /api/v1/results HTTP/1.1", host: "ara.domain.tld"
2020/06/26 19:49:56 [error] 3336#0: *35957 client intended to send too large body: 1481269 bytes, client: 10.168.43.105, server: ara.domain.tld, request: "POST /api/v1/results HTTP/1.1", host: "ara.domain.tld"
2020/06/26 19:50:00 [error] 3336#0: *35958 client intended to send too large body: 1477393 bytes, client: 10.168.43.105, server: ara.domain.tld, request: "POST /api/v1/results HTTP/1.1", host: "ara.domain.tld"
```

### PR DESCRIPTION
**Pull request by Thulium-Drake**
Allows to configure the client maximum body for Nginx.

Fixes #1

### CODE REVIEW

File: roles/ara_frontend_nginx/defaults/main.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  ARA Records Ansible is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with ARA Records Ansible. If not, see <http://www.gnu.org/licenses/>.
```
Comment (dmsimard) [inline roles/ara_frontend_nginx/defaults/main.yaml:17]:
```suggestion
---
# Copyright (c) 2021 The ARA Records Ansible authors
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

:point_up_2: is shorter, inclusive of Red Hat but also other authors -- I haven't got around to updating it everywhere but it's what I have been using for new files (like https://github.com/ansible-community/ara/blob/master/ara/ui/templatetags/diff_result.py#L1-L2)

**Comment by dmsimard** [comment]
Just a minor fix before we can merge, otherwise looks good.

File: roles/ara_frontend_nginx/defaults/main.yaml (hunk 1 above)
Comment (Thulium-Drake) [reply roles/ara_frontend_nginx/defaults/main.yaml:17]: Ah, cool, whenever I do change one of these, I'll try not to forget this ;-)

**Comment by dmsimard** [comment]
Cool, thanks !

**Pull request merged**